matchzoo/engine/base_model.py

```diff
@@ -98,7 +113,8 @@ def fit(
             x: typing.Union[np.ndarray, typing.List[np.ndarray]],
             y: np.ndarray,
             batch_size: int = 128,
-            epochs: int = 1
+            epochs: int = 1,
```

Comment:
I guess we should configure default `epoch`  as a larger value? something like 20 as default, if user forget to set `epochs` when fitting a model, at least it will not end-up with 1 epoch.

Reply:
`epoch` defaults to 1 is better, since not setting it means "I am not thinking about multiple epochs." Also, `keras` defaults to 1, and I kept it consistent (also for `batch_size`).

Reply:
okay